articles/active-directory/develop/msal-net-client-assertions.md

```diff
@@ -116,30 +123,30 @@ string Encode(byte[] arg)
     return s;
 }
 
-string GetSignedClientAssertion()
+static string GetSignedClientAssertion(string tenantId, string clientId)
 {
-    //Signing with SHA-256
-    string rsaSha256Signature = "http://www.w3.org/2001/04/xmldsig-more#rsa-sha256";
-     X509Certificate2 certificate = new X509Certificate2("Certificate.pfx", "Password", X509KeyStorageFlags.EphemeralKeySet);
+    // It's not recommended to hardcode a password or a file location. Get the X509Certificate2 from somewhere save.
+    X509Certificate2 certificate = new X509Certificate2("Certificate.pfx", "Password", X509KeyStorageFlags.EphemeralKeySet);
```

Comment:
Let's make the cert a parameter in the `GetSignedClientAssertion` method and add a comment on the method that the cert should be loaded from a secure location

Reply:
Great idea!